.github/workflows/chromatic.yml

```diff
@@ -4,8 +4,10 @@ name: "Chromatic"
 # Event for the workflow
 on:
   push:
+    branches:
+      - develop
   pull_request:
-    types: [opened, reopened]
+    types: [opened, reopened, synchronize]
```

Comment:
am confused by this change :( 
I thought we were opting for a comment based trigger instead?

Reply:
I dont think this fits in our mental model for a PR leh, eg when we have stacked PRs, design should still be able to comment on that changes even if they are not directly pushed to develop!

Reply:
we are, can close this first if you want. 

> when we have stacked PRs, design should still be able to comment on that changes even if they are not directly pushed to develop

this will work still cos when the PR is synced, it will trigger the workflow